inputs/orchestrator_sources.json

```diff
@@ -0,0 +1,33 @@
+{
+  "apiVersion": "build.openshift.io/v1",
+  "kind": "BuildConfig",
+  "metadata": {
+    "name": "{{NAME}}",
+    "labels": {
+      "is_autorebuild": "false"
+    }
+  },
+  "spec": {
+    "output": {
+      "to": {
+        "kind": "DockerImage",
+        "name": "{{REGISTRY_URI}}/{{OUTPUT_IMAGE_TAG}}"
+      }
+    },
+    "strategy": {
+      "type": "Custom",
+      "customStrategy": {
+        "from": {
+          "kind": "DockerImage",
+          "name": "{{BUILD_IMAGE}}"
+        },
+        "exposeDockerSocket": true,
+        "env": [{
+          "name": "ATOMIC_REACTOR_PLUGINS",
```

Comment:
we don't need this, we are deleting it anyway
as plugins are rendered based on user_params => plugins_configuration

so since it is new json, I'd just omit that

Reply:
Removed